app/models/jurisdiction.rb

```diff
@@ -3,6 +3,8 @@ class Jurisdiction < ActiveRecord::Base
   has_many :office_jurisdictions
   has_many :offices, through: :office_jurisdictions
 
+  has_many :business_entities
```

Comment:
Yeah, I think we won't be needing this, because we don't need to look at all business entity codes for a jurisdiction.